CHANGES_NEXT_RELEASE

```diff
@@ -1,2 +1,3 @@
 - Fix transport in autoprovision device depending on binding [#235]
-- Allow get list of commands without sending measures (empty payload) [#234]
\ No newline at end of file
+- Allow get list of commands without sending measures (empty payload) [#233]
```

Comment:
It seems you have changed 233 in the wrong line... L2 should change should be rolled-back, L3 should change 250 to 233, as I mention in a previos comment.

Reply:
Fixed in 457a1d8d4f910b14a887f6903803956374a834a5